app/javascript/components/pages/AboutUs.js

```diff
@@ -1,13 +1,188 @@
-import React from "react"
-import PropTypes from "prop-types"
+import React from "react";
+import { Button, Col, Form, FormGroup, Input, Label, Row } from "reactstrap";
+import Brian from "../assets/Brian.png";
+import chuck from "../assets/chuck.png";
+import matt from "../assets/matt.png";
```

Comment:
These should have the same casing. I'm thinking lowercase make sense since it's the image is not a class. Not a blocker, but you may want to take care of it now since its an easy change.

[FOLLOW]

Reply:
Same casing was applied! Actually meant to change that prior to the push.